src/Stats/index.jsx

```diff
@@ -51,12 +53,16 @@ export default function Stats() {
       <ScrollView style={styles.deviceBody}>
         <View style={{ marginBottom: 20 }}>
           <View style={{ flexDirection: 'row', justifyContent: 'space-between' }}>
-            <FilterBtn />
+            <FilterBtn
+              transFilter={transFilter}
```

Comment:
lets call this props `currentFilter` or `appliedFilter`

Reply:
fixed